products/compute/terraform.yaml

```diff
@@ -345,7 +345,7 @@ overrides: !ruby/object:Overrides::ResourceOverrides
       Adds existing resource policies to a disk. You can only add one policy
       which will be applied to this disk for scheduling snapshot creation.
 
-      ~> **Note:** This resource does not support zonal disks (`google_compute_disk`).
+      ~> **Note:** This resource does not support zonal disks (`google_compute_disk`). For zonal disks, please refer to https://www.terraform.io/docs/providers/google/r/compute_disk_resource_policy_attachment.html 
```

Comment:
it might be worth using 

```
... For zonal disks, use [`google_compute_disk_resource_policy_attachment`](https://www.terraform.io/docs/providers/google/r/compute_disk_resource_policy_attachment.html) instead.

```

instead

Reply:
thanks ! 

I changed it for the 2 lines.